hunting/aws/queries/iam_unusual_default_aviatrix_role_activity.toml

```diff
@@ -0,0 +1,27 @@
+[hunt]
+author = "Elastic"
+description = """
+This hunting query identifies unusual activity related to the default Aviatrix role in AWS CloudTrail logs. The Aviatrix role is a default role created by the Aviatrix Controller to manage AWS resources. Unusual activity may indicate unauthorized access or misuse of the Aviatrix role, potentially leading to data exfiltration, privilege escalation, or other security incidents.
+"""
+integration = ["aws.cloudtrail"]
+uuid = "9fe48b6e-d83a-11ef-84a6-f661ea17fbcd"
+name = "IAM Unusual Default Aviatrix Role Activity"
+language = ["ES|QL"]
+license = "Elastic License v2"
+notes = [
+"Review the `aws.cloudtrail.user_identity.arn` field to identify the Aviatrix role.",
+"Review the `aws.cloudtrail.resources.arn` field to identify the EC2 instance associated with the activity.",
+"Review security group and network ACL configurations for the EC2 instance to ensure they are not overly permissive or allow unauthorized access.",
+"Using the EC2 instance, pivot into VPC Flow Logs to identify network traffic patterns and potential lateral movement.",
+]
+mitre = ['T1078.004']
+query = ['''
+from logs-aws.cloudtrail-*
+| where @timestamp > now() - 14 day
+| where event.dataset == "aws.cloudtrail"
+    and aws.cloudtrail.user_identity.type == "AssumedRole"
+    and aws.cloudtrail.user_identity.arn like "*aviatrix-role*"
+| stats activity_counts = count(*) by event.provider, event.action, aws.cloudtrail.user_identity.arn
+| where activity_counts < 10
```

Comment:
it could be that the activity is new, maybe some unique/distinct count on the source of it's usage (like used from 2 or more sources) to increase suspicion ?